Tasks/JavaToolInstallerV0/task.loc.json

```diff
@@ -13,7 +13,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 0,
-    "Minor": 177,
+    "Minor": 179,
```

Comment:
Could you please change it to 180 since this task migration is planned to be rolled out at 180 sprint?

Reply:
Bumped version